src/components/TransactionItemRow/index.tsx

```diff
@@ -206,9 +212,9 @@ function TransactionItemRow({
                 <View style={[StyleUtils.getReportTableColumnStyles(CONST.SEARCH.TABLE_COLUMNS.FROM)]}>
                     {!!transactionItem.from && (
                         <UserInfoCell
-                            accountID={transactionItem.from.accountID}
-                            avatar={transactionItem.from.avatar}
-                            displayName={transactionItem.from.displayName ?? ''}
+                            accountID={transactionItem?.from.accountID}
```

Comment:
At the top, we already have a condition that checks if from exists, so I'm not sure if ? is necessary here.

`                    {!!transactionItem.from && (
`